### ISSUE TITLE
Support Python 2.6

### ISSUE THREAD

**Issue by RyanMillerC** [RyanMillerC/clogging#9]
Conlog does not currently work on Python 2.6. 

* Add support for Python 2.6.

**Comment by hugovk** [comment]
Is Python 2.6 something you specifically need?

Many projects are dropping it as it's been EOL since 2013-10-29, it's no longer receiving security updates, and is becoming harder to support due to dependencies and tools no longer supporting it.

**Comment by RyanMillerC** [comment]
No, it's not something that I need right now. I made this issue so I wouldn't forget about it. 

Python 2.6 is shipped with RHEL 6 though, and I wanted to still be able to support the system native Python.

If it's easy to implement, I'll add it. If it looks like major rework to support, I will leave it alone.
